packaging/linux/build_binaries.sh

```diff
@@ -36,13 +36,19 @@ fi
 echo "-tags '$go_tags'"
 
 # Determine the LD flags.
-ldflags=""
+ldflags_client=""
+ldflags_kbfs=""
 if [ "$mode" != "production" ] ; then
   # The non-production build number is everything in the version after the hyphen.
   build_number="$(echo -n "$version" | sed 's/.*-//')"
-  ldflags="-X github.com/keybase/client/go/libkb.PrereleaseBuild=$build_number"
+  ldflags_client="-X github.com/keybase/client/go/libkb.PrereleaseBuild=$build_number"
+  kbfs_path="$here/../../../kbfs"
```

Comment:
Yes, line 103 makes a symlink, and that needs to stay where it is. But I think we can move `$kbfs_repo` from line 102 up to the top and use it in both places. I think `$this_repo` is always going to be absolute, so the path should be valid for both.